src/tests/gov/nasa/jpf/test/D.java

```diff
@@ -0,0 +1,9 @@
+// src/tests/gov/nasa/jpf/test/D.java
+package gov.nasa.jpf;
+
+public class D {
+    // Non-static method
+    public void m() {
```

Comment:
This method should be static but then be compiled separately as a non-static method after an edit (which is made by `build.gradle`).